auth/data.js

```diff
@@ -22,7 +22,7 @@ var Client = base.Entity.configure({
      * - lastRotated      // Last time accessToken was reset
      * (more properties may be added in the future)
      */
-    details:        base.Entity.types.JSON
+    details:        Entity.types.JSON
   },
   context:          ['resolver']
```

Comment:
We could probably get rid of this circlar dependency... and take it as a parameter for `.json`